test/new-e2e/tests/agent-platform/common/file-manager/unix.go

```diff
@@ -23,21 +25,66 @@ func NewUnixFileManager(vmClient client.VM) *Unix {
 }
 
 // FileExists check if the file exists, return an error if it does not
-func (u *Unix) FileExists(path string) (string, error) {
-	return u.vmClient.ExecuteWithError(fmt.Sprintf("sudo stat %s", path))
+func (u *Unix) FileExists(path string) (bool, error) {
```

Comment:
All these methods were created before the implementation of the file manager in `e2e/client/vm_client.go`. Now I think that's fine to remove this `unix.go` file and to use the same client for both `windows` and `unix` since the implementation on `e2e/client/vm_client.go` should support both

Reply:
Unfortunately we can't because the SSH/SFTP connection on Linux machines is not running as root, so some file operations fail with access denied.